packages/integrations/preact/src/index.ts

```diff
@@ -20,6 +20,7 @@ export default function ({ include, exclude, compat }: Options = {}): AstroInteg
 		hooks: {
 			'astro:config:setup': ({ addRenderer, updateConfig, command }) => {
 				const preactPlugin = preact({
+					reactAliasesEnabled: compat,
```

Comment:
```suggestion
					reactAliasesEnabled: compat ?? false,
```
From the typedocs, the default for `reactAliasesEnabled` is `true`. It would be problematic for projects using both react and preact components, such as several of our test fixtures.

Reply:
Ah, did this from my phone, didn't notice `compat` isn't always set. Cheers